### DISCUSSION TITLE
Agenda view: show the next 28 days (4 weeks) events in calendar

### DISCUSSION THREAD

**Discussion by serrq** [tutao/tutanota#7830] [Ideas]
Hey, what is this?

I want see my next 4 weeks appointments in a list manner (all).

![1000125366](https://github.com/user-attachments/assets/5f09df0d-eb7a-45a9-8226-245b51618e1c)

**Comment by marcosdiasalves** [comment]
Please do not mess with this view. If you want to change anything, create a new view (called Schedule or something like that).
